### ISSUE TITLE
[basic.lookup.unqual]/6 is hard to parse

### ISSUE THREAD

**Issue by jwakely** [cplusplus/draft#451]
> A name used in the definition of a function following the function’s _declarator-id_ that is a member of namespace `N`

What does "that is a member of namespace `N`" refer to, the name or the function?

Normal English grammar would suggest it refers to the name, but the rest of the paragraph only seems to make sense if it refers to the function.

If I'm reading it correctly (which I'm still not sure about) "following the function's declarator-id" refers to where the name is used, but "is a member of namespace N" refers to the function, and yet neither of those clauses is adjacent to the thing they refer to!

This is like "The grass, in my garden, which is green, behind my house, ..."

**Comment by zygoloid** [comment]
Maybe something like "In the definition of a function that is a member of namespace `N`, a name used after the function's *declarator-id* is looked up in the following [...]" would be more natural.
